demo/src/screens/componentScreens/MarqueeScreen.tsx

```diff
@@ -0,0 +1,34 @@
+import _ from 'lodash';
+import React, {Component} from 'react';
+import {View, Text, Marquee} from 'react-native-ui-lib';
+
+export default class MarqueeScreen extends Component { 
+  renderRow() {
+    return (
+      <View row>
+        {_.times(8, i => {
+          return <View style={{backgroundColor: '#FF10F0', width: 20, height: 6, marginRight: 10}}/>;
+        })}
+      </View>
+    );
+  }
+  
+  render() {
+    return (
+      <View flex padding-20 style={{overflow: 'hidden'}}>
+        <Text text50 grey10 marginB-20>Marquee</Text>
+        <Marquee duration={2000}>
```

Comment:
The component should inherit its parent bounds, in this case for example, the component should have `padding-20`

Reply:
ok

Reply:
Now the sliding view is in the right width, but it's still running all over the screen width.
You'll be able to see what I'm talking about if you'll change the screen padding to 120 for example, then the time you wait after it gets out of the screen and until it enters again is too long.

Reply:
I'm not going to adjust the animation to the container. The user will have to pass the 'width' prop for that

Reply:
It doesn't work, try the following example: 
```
        <View bg-red60 width={50}>
          <Marquee width={50}>
            <Text>
              Test
            </Text>
          </Marquee>
        </View>
```

Reply:
Of course it won't work - if you wrap the component in a container of width 50 it can't move in it, they both have the same size... Give the container width 150 and see the Marqee move in it. The width passed to the Marquee is for calculating the animation (default is screen width)

Reply:
I meant that I do see the animation inside the container, but the time I should wait after it gets out of the container and until it enters again is too long because the animation runs all over the screen width (even though we can't see it).
The user can't control that, we have to handle it from the component. 
The animation should be 'flexed' in its container.

Reply:
When passing the 'width' prop the user controls the animation - the animation will run by the width passed and not the screen width. For example, if a user has a container with 20 paddings he should pass a width of screenWidth - 40.

Reply:
You know what, I'm closing this PR till I get specific requirements.

Reply:
I don't think we have to close it, it is well implemented.
We can ask Diana about this small issue so we can all get on board.

Reply:
@lidord-wix Since this is a component made without any ux requirements or a real use case I can work with, I'm implementing a basic full-screen-width component. Your comments for extra requirements are missing the point of this small guild-week task and are not in its scope. I do not wish to continue implementing requirements that might not be necessary.

Reply:
It is not mentioned in the ticket (and maybe that's the problem) but the first use case is the AudioPlayer component, the title of the audio player should use marquee when its title is too long.

Reply:
Please attach the guidelines to the ticket